flake.nix

```diff
@@ -0,0 +1,41 @@
+{
+	description = "FUTO's MDNS library";
+
+	inputs = {
+		nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+	};
+
+	outputs = {
+		self,
+		nixpkgs,
+	}: let
+		system = "x86_64-linux";
+		pkgs = nixpkgs.legacyPackages.${system};
+
+		getArch = system: nix2Net."${system}";
+		nix2Net = {
+			x86_64-linux = "linux-x64";
+		};
```

Comment:
There is _theoretically_ a better way to do this. It involves finding some magic undeclared variables inside [this bash script](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/dotnet/build-dotnet-module/hooks/dotnet-install-hook.sh) and it's kind of a pain in the ass. I've half a mind to leave this here in protest against violating nixpkgs code guidelines, but I think I may have found a workaround. I'll test it a bit more though to make sure it's not just black magic.

Reply:
Fixed in [8e0221e](https://github.com/futo-org/FUTO.MDNS/pull/1/commits/8e0221e73125d4f549ec7b334bbda494b6b13e18)